tsconfig.base.esm.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "module": "ES6",
+    "moduleResolution": "node",
```

Comment:
`moduleResolution: "node"` is the default value IIRC, hence we can skip it

Reply:
- [ ] skip module resolution